src/index.js

```diff
@@ -22,12 +22,21 @@ const injectDOM = function CozyBarInjectDOM ({lang, appName, iconPath}) {
   const appNode = document.querySelector('[role=application]')
```

Comment:
As the BAR is dedicated to be used in cozy's apps, I assume we can rely on the `[role=application]` tag, as this one is in our base template.

Eventually, we can add an option in the BAR's `init` method, tht allow any developer to specify a CSS selector for its app's root tag, and fallback to `[role=application]` by default?